src/core_plugins/kibana/public/discover/directives/_no_results.scss

```diff
@@ -0,0 +1,13 @@
+.dscNoResults {
+  max-width: 1000px;
+}
+
+// SASSTODO: seems like this class is not used anywhere
+.discoverNoResultsShardFailure {
+  display: block;
+
+  .euiFlexItem:last-child {
```

Comment:
Apply a new selector to the template on top. We never want to overwrite `.eui` classes in Kibana, even if its a deep selection like this.

```<EuiFlexItem className="myThing">```